dali/dfu/dfuutil.cpp

```diff
@@ -832,7 +845,14 @@ class CFileCloner
         }
         else
         {
+            IPropertyTree *dstClonedFDesc = dfile->queryAttributes().queryPropTree("cloneFromFDesc");
+            IPropertyTree *srcClonedFDesc = srcfdesc->queryProperties().queryPropTree("cloneFromFDesc");
+            if ((dstClonedFDesc && !srcClonedFDesc) || (!srcClonedFDesc && dstClonedFDesc) ||
+                (areMatchingPTrees(dstClonedFDesc, srcClonedFDesc)))
+                return true;
             StringBuffer s;
+            if (checkValueChanged(dfile->queryAttributes().queryProp("@cloneFrom"), srcdali->endpoint().getUrlStr(s).str()))
```

Comment:
Is this check now duplicated?

Reply:
I guess yes, if both cloneFromFDesc's exists, should check them, but avoid other tests.
I'll change code.

Reply:
oh, line was duplicated!
Well it made me look at an optimization anyway